reference/modules/logging.md

```diff
@@ -0,0 +1,144 @@
+# Logging
+
+What is logging and why do I want to use it
```

Comment:
I think we still need some content here.

Reply:
added